modules/apps/portal-language/portal-language-override-web/src/main/java/com/liferay/portal/language/override/web/internal/display/context/ViewManagementToolbarDisplayContext.java

```diff
@@ -97,9 +104,10 @@ public CreationMenu getCreationMenu() {
 					).buildString());
 				labelItem.setDismissible(true);
 				labelItem.setLabel(
-					LanguageUtil.get(httpServletRequest, "override"));
-			}
-		).build();
+					LanguageUtil.get(httpServletRequest, navigation));
```

Comment:
It will be more clear if we add `Override` to the filter label,  like `Override: Any Language` instead of just `Any Language`.

What do you think?

Reply:
I had thought about that, but I also changed the general filter dropdown label to indicate that we are filtering by override: https://github.com/drewbrokke/liferay-portal/pull/1382/commits/8674392eede5d96ec4fa47ffd85a25654fa2db6c. Do you think that could be enough? I'm open to either option.

Reply:
This label is shown when the filter is applied so the `Filter By Override` is not visible here. This is what it looks like in the UI
 
<img width="1270" alt="Screen Shot 2022-02-17 at 4 14 37 PM" src="https://user-images.githubusercontent.com/5168351/154606287-7b36d9bb-39e5-4bab-b298-bc959846640d.png">